solidity/contracts/vortex/VortexBurner.sol

```diff
@@ -0,0 +1,421 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.6.12;
+
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/math/Math.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+
+import "@bancor/token-governance/contracts/ITokenGovernance.sol";
+
+import "../converter/interfaces/IConverterRegistry.sol";
+import "../converter/interfaces/IConverter.sol";
+
+import "../utility/ContractRegistryClient.sol";
+import "../utility/Owned.sol";
+import "../utility/Utils.sol";
+import "../utility/TokenHolder.sol";
+import "../utility/ReentrancyGuard.sol";
+
+import "../INetworkSettings.sol";
+import "../IBancorNetwork.sol";
+
+/**
+ * @dev This contract provides any user to trigger a network fee burning event
+ */
+contract VortexBurner is Owned, Utils, ReentrancyGuard, ContractRegistryClient {
+    using SafeMath for uint256;
+    using Math for uint256;
+    using SafeERC20 for IERC20;
+
+    struct Strategy {
+        address[][] paths;
+        uint256[] amounts;
+        address[] govPath;
+    }
+
+    struct NetNetworkTokenConversionAmounts {
```

Comment:
seems a bit excessive to have this struct now that the `availableForBurning` function is removed, no?

Reply:
I agree. I'll refactor